### PR TITLE
Celsius farenheit convertation and yml test

### PR DESCRIPTION
**Pull request by daria-rus**
Convert_to_celsius.c convert celsius to farenheit. Add Action - test_sf.yml

### CODE REVIEW

File: celsius_farenheit/convert_to_celsius.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    int lower, upper, step;
+
+    lower = 0;
+    upper = 300;
+    step =20;
+    printf("%4s %7s\n", "fahr","celsius");
+    fahr=lower;
+    while(fahr <= upper){
+        celsius = (5.0/9.0)*(fahr - 32.0);
+        printf("%4.0f %7.1f\n",fahr,celsius);
```
Comment (tambel) [inline celsius_farenheit/convert_to_celsius.c:16]: what do `4.0` and `7.1` mean?

File: celsius_farenheit/temperature_converter.c [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# include <stdio.h>
+
+int main (){
+
+float c = 5.0/9.0;
+ for (int farh=0; farh<=300; farh+=20)
+    printf("%3d %6.1f\n", farh, c*(farh-32));
```
Comment (tambel) [inline celsius_farenheit/temperature_converter.c:7]: what do `3` and `6.1` mean?

**Review by tambel — COMMENTED** [review]
If you have two places that both use the same logic, why wouldn't you move that logic to a separate file and include it in your programs?

File: celsius_farenheit/convert_to_celsius.c (hunk 1 above)
Comment (daria-rus) [reply celsius_farenheit/convert_to_celsius.c:16]: 4.0f : 
1 argument of printf function is float, 4- 4 characters in whole part, .0 - 0 characters in decimal part to print

File: celsius_farenheit/temperature_converter.c (hunk 2 above)
Comment (daria-rus) [reply celsius_farenheit/temperature_converter.c:9]: see comment below

**Comment by tambel** [comment]
@daria-rus Please use better naming for your GHA tests
image

File: celsius_farenheit/convert_to_celsius.c (hunk 1 above)
Comment (tambel) [reply celsius_farenheit/convert_to_celsius.c:16]: got it.

**Review by tambel — APPROVED** [review]

**Comment by daria-rus** [comment]
use pullrequest io_func

**Pull request closed without merging**